### PR TITLE
Blink rogue integration

### PR DESCRIPTION
**Pull request by sergiitk**
#### What's this PR do?
Ths PR introduces basic API wrapper for [DoSomething/blink](https://github.com/DoSomething/blink).

The following event endpoints are supported:

```php
// POST https://blink.dosomething.org/api/v1/events/user-signup
// Creates user signup to a campaign.
$blinkRestClient->userSignup($rogueSignupPayload);

// POST https://blink.dosomething.org/api/v1/events/user-signup-post
// Creates user signup post (reportback) to a campaign.
$blinkRestClient->userSignupPost($rogueSignupPostPayload);
```

#### How should this be manually tested?
`./vendor/bin/phpunit`

#### Any background context you want to provide?
This is a part of Rogue -> Blink -> customer.io integration pipeline.

#### Checklist
- [x] Tests added for new features/bug fixes.
- [x] Tested with blink staging

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,6 +55,44 @@ $northstar->withToken($accessToken)->get('v1/profile');
 
 ```
 
+**Blink**
+
+This package includes basic API wrapper for [DoSomething/blink](https://github.com/DoSomething/blink).
+
+Usage is similar to the example above with an exception that it doesn't require using bridge class,
+and rely on HTTP Basic Authentication with by HTTP header instead.
```
Comment (DFurnes) [inline README.md]: ✏️ `s/with by/with`

**Review by DFurnes — APPROVED** [review]
Looks good to me! One little typo.

File: README.md (hunk 1 above)
Comment (sergiitk) [reply README.md]: 👀

**Review by DFurnes — APPROVED** [review]

**Review by sbsmith86 — APPROVED** [review]
looks good to me

**Pull request merged**